canonical-web-and-design/latest-news
# Add script to package

## Done

- Added the latest news JS to the package
- Updated the README with information about usage and development

## QA

- Clone this repo
- Run `yarn && yarn build`
- Run `yarn serve`
- Check the URL output by the serve command in your browser
- Check that there are blog posts displayed similar to the ubuntu.com homepage

## Issue / Card

Fixes #7, #8
